src/main/java/com/criticalsoftware/giveaway/AnnouncementResponse.java

```diff
@@ -0,0 +1,76 @@
+package com.criticalsoftware.giveaway;
+
+import com.criticalsoftware.Product;
+import org.bson.types.ObjectId;
+
+import java.time.LocalDateTime;
+
+//Announcement Response
+public class AnnouncementResponse {
+
+    private String id;
+    private Product product;
+    private LocalDateTime date;
+    private ObjectId userDonorId;
+    private ObjectId userDoneeId;
+
+
+    public AnnouncementResponse(String id, Product product, String userDonorId,String userDoneeId) {
+
```

Comment:
Why do you need this empty constructor?

Reply:
If I remove the constructor the code will show errors